pkgs/top-level/python-packages.nix

```diff
@@ -14909,6 +14909,22 @@ in {
     };
   };
 
+  pint = buildPythonPackage rec {
+    name = "pint-${version}";
+    version = "0.7.2";
+
+    meta = {
+      description = "Physical quantities module";
+      license = licenses.bsd3;
+    };
+
+    src = pkgs.fetchFromGitHub {
```

Comment:
Having packages closer to the source makes indeed more sense and is something which I think we eventually should strive for. Sometimes we do use GitHub packages when the distributed archives don't include tests. 

There isn't any written policy and so what I write here is my own opinion on the matter.
What I think is most important is that we use the exact same source as is used when doing a `pip install`. Aside from that, I would like to keep the expressions as uniform as possible, because it makes it easier to maintain as well as to make larger changes, like (future) methods to automate some of the packaging.